src/frontend/src/lib/components/auth/ButtonAuthenticateWithLicense.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+	import ButtonAuthenticate from '$lib/components/ui/ButtonAuthenticate.svelte';
+	import { signIn } from '$lib/services/auth.services';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+
+	export let fullWidth = false;
+	export let licenseAlignment: 'inherit' | 'center' = 'inherit';
+</script>
+
+<div class="ali flex w-full flex-col">
+	<ButtonAuthenticate on:click={async () => await signIn({})} {fullWidth} />
```

Comment:
it seemed like I could include the `signIn` function here directly instead of bubbling `on:click`. Any argument against it?

Reply:
`ButtonAuthenticate` is a `ui` component so I would rather keep the `signIn` feature out of it if that makes sense?